### PR TITLE
feat(team): add chris hawkins

### PR DESCRIPTION
**Pull request by chriswhawkins**
Fixes: #57

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,6 +231,16 @@
         </h4>
       </div>
     </li>
+
+    <li id="nschubach" class="team-member">
```
Comment (tborres) [inline index.html]: update the id to `chawkins`

File: index.html [hunk 2]
```diff
@@ -231,6 +231,16 @@
...
+      <img src="/assets/img/chawkins.jpg" alt="Nick Schubach" class="img-circle" />
```
Comment (tborres) [inline index.html]: Update `alt`

**Comment by tborres** [comment]
A couple changes and then it's ready to merge.

**Comment by chriswhawkins** [comment]
Yikes, thanks for catching that.

**Pull request merged**